entries/r/rakuten.com.json

```
@@ -1,5 +1,5 @@
 {
-  "Rakuten": {
+  "Rakuten US": {
     "domain": "rakuten.com",
     "url": "https://www.rakuten.com/",
```

Right, I'm not saying the databases are the same but the authentication systems are the same. That would mean that if they enabled as an option on one of the sites it would be enabled for the rest (excluding JP).
Therefore, please add the other domains in the `additional-domains` array. If you want to create a separate listing for Rakuten Japan that's also fine.
```suggestion
    "url": "https://www.rakuten.com/",
    "additional-domains": [
      "rakuten.ca",
      "rakuten.de",
      "rakuten.es",
      "rakuten.fr",
      "rakuten.com.tw"
    ],
```